src/endpoints/funding-status.js

```diff
@@ -49,33 +50,48 @@ const crawlOrderStatus = async () => {
 crawlOrderStatus().then(debug)
 
 module.exports.funding_status = async (event, context, callback) => {
-  context.callbackWaitsForEmptyEventLoop = false; 
+  context.callbackWaitsForEmptyEventLoop = false
 
-  const deltaTime = Math.abs( moment().diff(lastUpdate, 'seconds') )
+  try {
+    const deltaTime = Math.abs(moment().diff(lastUpdate, 'seconds'))
 
-  if(cacheTotalAmount === undefined || deltaTime > 60){
-    debug('update', deltaTime)
-    cacheTotalAmount = await crawlOrderStatus()
-    lastUpdate = new moment()
-  }
-  else{
-    debug('from cache')
-  }
-  
+    if (cacheTotalAmount === undefined || deltaTime > 60) {
+      debug('update', deltaTime)
+      cacheTotalAmount = await crawlOrderStatus()
+      lastUpdate = new moment()
+    } else {
+      debug('from cache')
+    }
 
-  return {
-    statusCode: 200,
-    headers: {
-      'Access-Control-Allow-Origin': '*', // Required for CORS support to work
-      'Access-Control-Allow-Credentials': true, // Required for cookies, authorization headers with HTTPS
-    },
-    body: JSON.stringify({
-      funding: cacheTotalAmount,
-      goal: 1,
-      accepting: true,
-      start: moment().startOf('isoWeek').toDate(),
-      end: moment().endOf('isoWeek').toDate(),
-      ts: moment()
-    })
+    return {
+      statusCode: 200,
+      headers: {
+        'Access-Control-Allow-Origin': '*', // Required for CORS support to work
+        'Access-Control-Allow-Credentials': true, // Required for cookies, authorization headers with HTTPS
+      },
+      body: JSON.stringify({
+        funding: cacheTotalAmount,
+        goal: 1,
+        accepting: true,
+        start: moment()
+          .startOf('isoWeek')
+          .toDate(),
+        end: moment()
+          .endOf('isoWeek')
+          .toDate(),
+        ts: moment(),
+      }),
+    }
+  } catch (e) {
+    return {
+      statusCode: 400,
+      headers: {
+        'Access-Control-Allow-Origin': '*', // Required for CORS support to work
+        'Access-Control-Allow-Credentials': true, // Required for cookies, authorization headers with HTTPS
+      },
+      body: JSON.stringify({
+        error: e.message,
```

Comment:
reply with generic error message(but debug/console log the real one). the Stripe errors can leak private data

Reply:
>and not all the error messages are suitable to be shown to the user
So I went this way because I thought the same authed user would be the one seeing the error and we needed a cheap & fair-enough solution.

Completely understand the security concerns. I'll make the change to use a generic error message.